golang/pghoard_postgres_command_go.go

```diff
@@ -120,15 +121,18 @@ func restore_command(url string, output string, xlog string) (int, error) {
 			}
 			output_path = path.Join(cwd, output)
 		}
-		// if file "<xlog>.pghoard.prefetch" exists, just move it to destination
-		xlogPrefetchPath := path.Join(path.Dir(output_path), xlog+".pghoard.prefetch")
-		_, err = os.Stat(xlogPrefetchPath)
-		if err == nil {
-			err := os.Rename(xlogPrefetchPath, output_path)
-			if err != nil {
-				return EXIT_ABORT, err
+		xlogNameRe := regexp.MustCompile(`^[A-F0-9]{24}$`)
```

Comment:
Maybe forget the RE for timeline file? E.g, `00000002.history`?

Reply:
As I remember we don't prefetch history files.
At least there shouldn't be many history files and fallback to HTTP call should work.

Reply:
We both and only prefetch timeline (i.e. history) and xlog files though.

Reply:
Extend regexp to `^([A-F0-9]{24}|[A-F0-9]{8}\.history)$`